CoffeeChat/SceneDelegate.swift

```diff
@@ -17,7 +17,12 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
         // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
         // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
-        guard let _ = (scene as? UIWindowScene) else { return }
+        guard let scene = scene as? UIWindowScene else { return }
+
+        let window = UIWindow(windowScene: scene)
+        window.rootViewController = InterestsViewController()
```

Comment:
Should show `loginVC` first. Check out what's on master.

Reply:
^ I think rebasing / merging will be slightly annoying lol

Reply:
also, you can simply keep what haiying has for now and I will add your view controller to my page controller later and handle the navigation logic between sign in and the onboarding screens!